packages/block-editor/src/hooks/use-bindings-attributes.js

```diff
@@ -100,36 +99,24 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 	( BlockEdit ) => ( props ) => {
 		const registry = useRegistry();
 		const blockContext = useContext( BlockContext );
-		const sources = useSelect( ( select ) =>
-			unlock( select( blocksStore ) ).getAllBlockBindingsSources()
-		);
+		const sources = getBlockBindingsSources();
 		const { name, clientId, context, setAttributes } = props;
-		const blockBindings = useMemo(
-			() =>
-				replacePatternOverrideDefaultBindings(
+		const { blockBindings, blockBindingsBySource, updatedContext } =
+			useMemo( () => {
+				const _blockBindings = replacePatternOverrideDefaultBindings(
 					name,
 					props.attributes.metadata?.bindings
-				),
-			[ props.attributes.metadata?.bindings, name ]
-		);
+				);
+				const _updatedContext = {};
```

Comment:
Are changes in this file still needed after https://github.com/WordPress/gutenberg/pull/67523? The PR might require a refresh anyway.

Reply:
You're right. I think they are not needed anymore. I've started exploring how it could look like [here](https://github.com/WordPress/gutenberg/pull/67885). I didn't want to add more noise here, but I can do a proper rebase once it is clear the path forward.